lib/AST/CASTBridging.cpp

```diff
@@ -111,185 +123,187 @@ void SwiftDiagnostic_fixItReplace(BridgedDiagnostic diagPtr,
 }
 
 /// Finish the given diagnostic and emit it.
-void SwiftDiagnostic_finish(BridgedDiagnostic diagPtr) {
+void Diagnostic_finish(BridgedDiagnostic diagPtr) {
   BridgedDiagnosticImpl *impl = static_cast<BridgedDiagnosticImpl *>(diagPtr);
   delete impl;
 }
 
 BridgedIdentifier
-SwiftASTContext_getIdentifier(void *ctx, const unsigned char *_Nullable str,
-                              long len) {
-  return const_cast<void *>(
-      static_cast<ASTContext *>(ctx)
-          ->getIdentifier(
-              StringRef{reinterpret_cast<const char *>(str), size_t(len)})
-          .getAsOpaquePointer());
-}
-
-void *SwiftImportDecl_create(void *ctx, void *dc, void *importLoc, char kind,
-                             void *kindLoc, BridgedArrayRef path,
-                             BridgedArrayRef pathLocs) {
-  assert(path.numElements == pathLocs.numElements);
-  ASTContext &Context = *static_cast<ASTContext *>(ctx);
+ASTContext_getIdentifier(BridgedASTContext cContext, BridgedString cStr) {
+  return { convertASTContext(cContext)
+          .getIdentifier(convertString(cStr))
```

Comment:
How does this work? Does C++ also have unresolved member access?

Reply:
I imagine I didn't run clang format and the indentation is throwing you off. This is just `convertASTContext(cContext).getIdentifier(convertString(cStr))`

Reply:
Ah, I thought there was a comma here. Makes sense.